cmd/operator/main.go

```diff
@@ -32,13 +33,17 @@ func main() {
 		namespace       string
 		metricsAddr     string
 		healthProbeAddr string
+		images          k8sflag.MapStringString
+		versions        k8sflag.MapStringString
 
 		setupLog = ctrl.Log.WithName("setup")
 	)
 
 	flag.StringVar(&namespace, "namespace", "default", "The namespace in which the operator runs")
 	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
 	flag.StringVar(&healthProbeAddr, "health-probe-bind-address", ":8081", "The address the health probe endpoint binds to.")
+	flag.Var(&images, "images", "Images to use for containers managed by the observability-operator.")
```

Comment:
I think it may help in the longer run to make use of `releatedImages` in cluster-service-version. see: https://docs.openshift.com/container-platform/4.14/operators/operator_sdk/osdk-generating-csvs.html#olm-enabling-operator-for-restricted-network_osdk-generating-csvs 

See also: https://github.com/sustainable-computing-io/kepler-operator/pull/242/files#diff-3492afe4e3cd6b74bb7970b8a215ba867f91984594da0cd558f5fe8dd03bfc6d where kepler-operator uses relatedImages to initialise the `kepler` image to be used.

Reply:
Using relatedImages seem the preferred approach. What are the current limitations that prevent us from doing so instead?

Reply:
We will use `relatedImages`. This PR here is just the plumbing for that. Getting an env var vs. explicit passing via cli flags is imho an implementation detail.
We will still use the `relatedImages` CSV field. The main reason this isn't part of the PR for now is that our internal build tooling atomates setting `relatedImages` but afaiu only if its not already present in the CSV. We can always add that to the CSV generation here at a later stage though.